client/src/plus/deep-dives/overview/index.scss

```diff
@@ -6,196 +6,167 @@
 @use "../../../ui/theme/index" as theme;
 
 .page-content-container.plus {
+  background-color: theme.$mdn-neutral050;
   max-width: inherit;
-  padding: layout.$base-spacing 0;
+  padding: 0;
+
+  @media #{layout.$mq-tablet-and-up} {
+    margin-top: 24px;
+  }
 }
 
-.overview-hero {
-  color: theme.$text-color-inverted;
-  overflow: hidden;
-  padding: 0;
-  position: relative;
+.deep-dives-overview {
+  background-color: theme.$white;
+  margin: 0 auto;
+  max-width: 600px;
 
   &::before {
-    background: transparent url("./assets/hero.svg") center center no-repeat;
+    background: transparent url("./assets/deep-dive-hero.svg") center center
```

Comment:
Change to `background: linear-gradient(0deg, #FFF 13%, rgba(255, 255, 255, 0.41) 25.05%, rgba(255, 255, 255, 0) 29.17%), transparent url(/static/media/deep-dive-hero.09268174.svg) 50% no-repeat;`